app/dashboard/templates/dashboard/mailbox_detail.html

```diff
@@ -64,6 +64,35 @@ <h1 class="h3">{{ mailbox.email }}
       </div>
       <!-- END Change email -->
 
+      {% if spf_available %}
+      <div class="card">
+        <form method="post">
+          <input type="hidden" name="form-name" value="force-spf">
+
+          <div class="card-body">
+            <div class="card-title">
+              Enforce SPF
+              <div class="small-text">
+                Block emails to reverse alias if sender is not validated by SPF,
```

Comment:
I think we'll need to explain more about SPF as this concept is not easy to understand for non-tech users :). But don't worry, I can take care of that later :).